lib/basedriver/helpers.js

```diff
@@ -303,6 +303,26 @@ function getSwipeTouchDuration (waitGesture) {
   return duration;
 }
 
+
+/**
+ * Recursively find all instances of the key 'inKey' and rename them 'outKey'
+ * @param {*} inp Any type of input
+ * @param {String} inKey The key name to replace
+ * @param {String} outKey The key name to replace it with
+ */
+function renameKey (inp, inKey, outKey) {
+  if (_.isArray(inp)) {
+    return inp.map((item) => renameKey(item, inKey, outKey));
+  } else if (_.isObject(inp)) {
```

Comment:
do you mean isPlainObject?

Reply:
yeah, probably. since we're dealing with a response, we should never have something which is not a plain object. maybe we should throw in that case since it'd be a programming error on the appium side.